src/main/java/frc/robot/RobotMap.java

```diff
@@ -28,24 +32,20 @@
 
     public interface LiftMap {
 
-        SendableSpeedController getMotor();
+        CANSparkMax getMotor();
 
         DoubleSolenoid getBrake();
 
-        SendableSpeedController getArmMotor();
-
-        Encoder getHeightEncoder();
+        SparkMaxCounter getHeightEncoder();
 
         DigitalInput getLowerLimit();
 
         DigitalInput getUpperLimit();
-
-        Potentiometer getManipAngle();
     }
 
     ManipulatorMap getManipulatorMap();
 
-    interface ManipulatorMap {
+    public interface ManipulatorMap {
```

Comment:
You shouldn't need to say that they're public, interface members are public by default.

Reply:
donezo 🐩